src/schema/nodes/list.ts

```diff
@@ -14,21 +14,35 @@
  * limitations under the License.
  */
 
-import { ObjectTypes } from '@manuscripts/json-schema'
+import { ListElement, ObjectTypes } from '@manuscripts/json-schema'
 import { NodeSpec } from 'prosemirror-model'
 
 import { buildElementClass } from '../../lib/attributes'
 import { ManuscriptNode } from '../types'
 
-export interface BulletListNode extends ManuscriptNode {
+export type JatsStyleType = NonNullable<ListElement['listStyleType']>
+
+export const JATS_HTML_LIST_STYLE_MAPPING: {
+  [key in JatsStyleType]: { style: string; type: string }
+} = {
+  simple: { style: 'none', type: 'ul' },
+  bullet: { style: 'disc', type: 'ul' },
+  order: { style: 'decimal', type: 'ol' },
+  'alpha-lower': { style: 'lower-alpha', type: 'ol' },
+  'alpha-upper': { style: 'upper-alpha', type: 'ol' },
+  'roman-lower': { style: 'lower-roman', type: 'ol' },
+  'roman-upper': { style: 'upper-roman', type: 'ol' },
+}
```

Comment:
Can we replace this with something like:
```
export const getListType = (style: string): 'ul' | 'ol' => {
...
}
```
The body can be a `switch` statement.

Let's not involve json-schema if possible.